aten/src/ATen/native/native_functions.yaml

```diff
@@ -7730,18 +7737,6 @@
   dispatch:
     CUDA: _amp_update_scale_cuda_
 
-- func: _cat(Tensor[] tensors, int dim=0) -> Tensor
-  dispatch:
-    CPU: _cat_cpu
-    CUDA: cat_cuda
-    QuantizedCPU: cat_quantized_cpu
-
-- func: _cat.out(Tensor[] tensors, int dim=0, *, Tensor(a!) out) -> Tensor(a!)
-  dispatch:
-    CPU: _cat_out_cpu
-    CUDA: cat_out_cuda
-    QuantizedCPU: cat_out_quantized_cpu
-
```

Comment:
@zou3519 do you happen to know the history behind why we had both an `at::cat` and `at::_cat`? I couldn't dig up much of a reason from git blame. Although it seems usefully to try to kill it as part of this structured kernel port.

Reply:
I don't know why we have both. I agree if we can kill `_cat` we should; there doesn't seem to be a need for both `_cat` and `cat` to exist